### ISSUE TITLE
RZ_A1H does not support bootloader

### ISSUE THREAD

**Issue by teetak01** [ARMmbed/mbed-os-example-pelion#100]
```
Building project mbed-os-example-pelion-RZ_A1H-ARM-TCP (RZ_A1H, ARMC6)
Scan: mbed-os-example-pelion
Using ROM regions bootloader, header, application in this build.
  Region bootloader: size 0xc000, offset 0x18000000
  Region header: size 0x70, offset 0x18010000
  Region application: size 0x7efc00, offset 0x18010400
Configuration error: Bootloader not supported on this target. RAM start not found in targets.json.
```

**Comment by ciarmcom** [comment]
*Internal Jira reference: https://jira.arm.com/browse/IOTPART-9034*

**Comment by virendrasinhchauhan** [comment]
I am having below error for CY8CPROTO_062_4343W board

> "Using ROM regions bootloader1, bootloader2, header, application in this build.
>   Region bootloader1: size 0x1400, offset 0x10000000
>   Region bootloader2: size 0xa400, offset 0x10002000
>   Region header: size 0x70, offset 0x1000c000
>   Region application: size 0x1f3c00, offset 0x1000c400
> Configuration error: Bootloader not supported on this target. RAM start not found in targets.json.
> Compile [  0.1%]: DebugModule.cpp
> [ERROR] arm-none-eabi-g++: error: CreateProcess: No such file or directory"


Any update for this issue?

**Comment by JanneKiiskila** [comment]
PR on Mbed OS https://github.com/ARMmbed/mbed-os/pull/12223

**Comment by teetak01** [comment]
Fixed as part of Mbed OS 5.15.2 

#127
